server/passport-setup.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const passport = require('passport');
+const GoogleStrategy = require('passport-google-oauth20');
+const LocalStrategy = require('passport-local').Strategy;
+const { Strategy: JwtStrategy, ExtractJwt } = require('passport-jwt');
+const User = require('./database').User;
+
+passport.serializeUser((user, done) => {
+  done(null, user.id);
+});
+
+passport.deserializeUser((id, done) => {
+  User.findById(id)
+    .then(user => done(null, user));
+});
+
+passport.use(new JwtStrategy({
+  jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+  secretOrKey: process.env.JWT_SECRET
+}, (payload, done) => {
+  return User.findOne({ id: payload.sub })
+    .then(user => done(null, user))
+    .catch(err => done(err));
+}));
+
+passport.use('register', new LocalStrategy({
+  usernameField: 'email',
+  passwordField: 'password'
+}, (email, password, done) => {
+  const where = { email };
+  return User.findOne({ where })
+    .then((user) => {
+      if (user) return done(null, false);
+      User.create({ email, password })
+        .then(user => done(null, user))
+        .catch({ name: 'SequelizeValidationError' }, err => {
+          done(err, false);
```

Comment:
```js
.catch({ name: 'SequelizeValidationError' }, err => done (err, false);
.catch ... ```

You don't need to follow 80 chars rule as the Holy scripture, (+/- 5%) 😅 
If you insist you can always extract 'SequelizeValidationError' to some common error variable than use it around.